pom.xml

```diff
@@ -47,7 +47,7 @@
 
         <deps.apache-sshd.version>0.12.0</deps.apache-sshd.version>
         <deps.jsch.version>0.1.52</deps.jsch.version>
-        <deps.ssh-credentials-plugin.version>1.13-SNAPSHOT</deps.ssh-credentials-plugin.version>
+        <deps.ssh-credentials-plugin.version>2.0-SNAPSHOT</deps.ssh-credentials-plugin.version>
```

Comment:
If you are only using the version in one place in the pom then it is a **total anti-pattern** to use properties to specify the version. The only time it makes sense to use properties to specify the version is when there are multiple dependencies or dependencies and plugins that need to be aligned on the same version. In those cases using a property is worth the risk. In all other cases using a property exposes the build and downstream builds to additional risks of instability.

Probably not something to address in this PR but rather in a separate PR.